test/system/src/main/java/io/pravega/test/system/framework/RemoteSequential.java

```diff
@@ -85,7 +84,7 @@
 
     private CompletableFuture<Void> waitForJobCompletion(final String jobId) {
         return Futures.loop(() -> isTestRunning(jobId),
-                () -> Futures.delayedFuture(Duration.ofSeconds(3), executorService),
+                () -> Futures.delayedFuture(Duration.ofSeconds(10), executorService),
```

Comment:
Could you argue why 10 is a better value than 3?

Reply:
All our tests are in the order of 10s of minutes. The intent of this change is to reduce the log clutter and network io by increasing the duration to 10 seconds.